druid/src/widget/label.rs

```diff
@@ -141,6 +141,15 @@ impl<T: Data> Label<T> {
         self
     }
 
+    /// Builder-style method to set the wrap width.
```

Comment:
So I think this is touching on a much deeper topic. With this, the word wrap is set explicitly and the widget layout follows, but if you look at something like Flutter, it's really the other way around; the text layout is wrapped to conform to whatever the widget layout resolved to. There's other moving parts to it, including the intrinsic widths etc., but I think if we are going to do "word wrap in a label" then maybe now is the time to hit those expectations.